backend/packages/Upgrade/test/unit/services/FeatureFlagService.test.ts

```diff
@@ -154,6 +155,14 @@ describe('Feature Flag Service Testing', () => {
       providers: [
         DataSource,
         FeatureFlagService,
+        {
+          provide: CacheService,
+          useValue: {
+            delCache: jest.fn().mockResolvedValue(undefined),
+            resetPrefixCache: jest.fn().mockResolvedValue(undefined),
+            wrap: jest.fn().mockImplementation((key, cb) => cb()),
+          }
```

Comment:
`Eslint` problem here. Need a `,`.
Do you have the Eslint extension installed?

Reply:
sure, the eslint job that runs on commit hook didn't catch  and it builds and passes everything, so I didn't see it. i think we are not all getting consistent linting action going on if vscode and our commit hooks aren't agreeing.